test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -6693,6 +6713,27 @@ def manager(self, cmd, obj_type, attrib=None, id=None, extend=None,
         :type logerr: bool
         :raises: PbsManagerError
         """
+        skipmsg = 'ncpus requested are not same as available'
+        if cmd == MGR_CMD_SET and id is not None:
+            if obj_type == NODE:
+                for cmom, momobj in self.moms.items():
+                    cpuset_nodes = []
+                    if momobj.is_cpuset_mom():
+                        vnodes = self.status(NODE)
```

Comment:
I think this should be `self.status(VNODE, id=cmom)` (aka only get vnodes of given natural node name)

Reply:
Hi @hirenvadalia , id=cmom runs /opt/pbs/bin/pbsnodes -s <server> -v <vnodename> this can be only used to fetch one vnode on a host not all the vnodes of a cpuset host. To get that we need to run pbsnodes -H <momhost>. Should we fix that in PTL ?

Reply:
Yes, please fix it in PTL (IIRC, that is long pending issue in PTL status())

Reply:
@hirenvadalia , If someone only wants to get information of one vnode on a cpusetmom, I don't know if somebody is using it. but it would break current functionality.

Reply:
@visheshh I am talking about pbsnodes -s server -H momhost, aka (only) all vnodes on given mom host... if that is working fine then we don't need to fix anything in PTL.
But `vnodes = self.status(NODE)` will give you all vnodes on all moms